docs/guides/gcp-private-service-connect-workspace.md

```diff
@@ -0,0 +1,311 @@
+  member  = "serviceAccount:${google_service_account.sa2.email}"
+}
+```
+
+After you’ve added Service Account to Databricks Accounts Console, please copy its name into `databricks_google_service_account` variable. If you prefer environment variables - `DATABRICKS_GOOGLE_SERVICE_ACCOUNT` is the one you’ll use instead. Please also copy Account ID into `databricks_account_id` variable.
+
+## Authenticate with Databricks account API
+
+Databricks account-level APIs can only be called by account owners and account admins, and can only be authenticated using Google-issued OIDC tokens. The simplest way to do this would be via [Google Cloud CLI](https://cloud.google.com/sdk/gcloud). The `gcloud` command is available after installing the SDK. Then run the following commands
+
+* `gcloud auth application-default login` to authorise your user with Google Cloud Platform. (If you want to use your [service account's credentials instead](https://cloud.google.com/docs/authentication/provide-credentials-adc#local-key), set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the path of the JSON file that contains your service account key)
+* `terraform init` to load Google and Databricks Terraform providers.
+* `terraform apply` to apply the configuration changes. Terraform will use your credential to impersonate the service account specified in `databricks_google_service_account` to call the Databricks account-level API.
+
+Alternatively, if you cannot use impersonation and [Application Default Credentials](https://cloud.google.com/docs/authentication/production) as configured by `gcloud`, consider using the service account key directly by passing it to `google_credentials` parameter (or `GOOGLE_CREDENTIALS` environment variable) to avoid using `gcloud`, impersonation, and ADC altogether. The content of this parameter must be either the path to `.json` file or the full JSON content of the Google service account key.
+
+## Provider initialization
+
+```hcl
+variable "databricks_account_id" {}
+variable "databricks_google_service_account" {}
+variable "google_project" {}
+variable "google_region" {}
+variable "google_zone" {}
+
+
+terraform {
+  required_providers {
+    databricks = {
+      source = "databricks/databricks"
+    }
+    google = {
+      source  = "hashicorp/google"
+      version = "4.47.0"
+    }
+  }
+}
+
+provider "google" {
+  project = var.google_project
+  region  = var.google_region
+  zone    = var.google_zone
+}
+
+// initialize provider in "accounts" mode to provision new workspace
+
+provider "databricks" {
+  alias                  = "accounts"
+  host                   = "https://accounts.gcp.databricks.com"
+  google_service_account = var.databricks_google_service_account
+  account_id             = var.databricks_account_id
+}
+
+data "google_client_openid_userinfo" "me" {
+}
+
+data "google_client_config" "current" {
+}
+
+resource "random_string" "suffix" {
+  special = false
+  upper   = false
+  length  = 6
+}
+```
```

Comment:
This block seems to be irrelevant to GCP PSC. Can we refer the https://src.dev.databricks.com/databricks/terraform-provider-databricks/-/blob/docs/guides/gcp-workspace.md#creating-a-gcp-service-account-for-databricks-provisioning pages instead of copy/paste?

Reply:
Done.